src/components/ThemeProvider/__stories__/ThemeProvider.stories.helpers.tsx

```diff
@@ -201,3 +201,48 @@ export const UsageGuidelinesThemeProvider = () => (
     ]}
   />
 );
+
+export const DescriptionWithLinkMondaySdkIntegration = () => (
+  <>
+    In case you are developing an external application for monday.com. You can use ThemeProvider in combination with the{" "}
+    <Link
+      href="https://developer.monday.com/apps/docs/mondayget#sample-context-objects-for-each-feature-type"
+      withoutSpacing
+    >
+      monday.com SDK
+    </Link>
+    , to apply monday.com <b>system</b> and <b>product</b> themes to your application. This will allow your application
+    to be consistent with the monday.com UI.
+  </>
+);
+
+export const MondaySdkIntegrationSourceCode = `
+import { ThemeProvider } from "monday-ui-react-core";
+import mondaySdk from "monday-sdk-js";
+
+const monday = mondaySdk();
+
+export const useGetContext = () => {
+  const [context, setContext] = useState({});
+  
+  useEffect(() => {
```

Comment:
BTW, I think we should suggest the SDK team provide a react wrapper for the SDK. it's a boilerplate code and could be easily hidden

Reply:
I actually believe that they should... It doesn't look good here
I'll ask, but don't want it to add additional delay to the task...

Reply:
https://monday.slack.com/archives/C0663Q9KCDN/p1705848528450079

Reply:
Boilerplate code reduced with Rami's help